src/patterns/utilities/single-property/color.hbs

```diff
@@ -73,44 +74,84 @@ description: These utilities control CSS properties related to color and backgro
+      <td class="sprk-u-FontWeight--bold">.sprk-u-BackgroundColor--gray-dark</td>
+      <td>Sets the background color to dark gray.</td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">.sprk-u-BackgroundColor--blue</td>
+      <td>Sets the background color to blue.</td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">.sprk-u-BackgroundColor--blue-tint-75</td>
+      <td>Sets the background color to blue tinted at 75%.</td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">.sprk-u-BackgroundColor--blue-tint-50</td>
+      <td>Sets the background color to blue tinted at 50%.</td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">.sprk-u-BackgroundColor--blue-tint-25</td>
+      <td>Sets the background color to blue tinted at 25%.</td>
     </tr>
     <tr>
       <td class="sprk-u-FontWeight--bold">.sprk-u-BackgroundColor--gray</td>
       <td>Sets the background color to gray.</td>
     </tr>
     <tr>
-      <td class="sprk-u-FontWeight--bold">.sprk-u-BackgroundColor--gray-tint-75</td>
-      <td>Sets the background color to the gray tinted at 75%.</td>
+      <td class="sprk-u-FontWeight--bold">.sprk-u-BackgroundColor--green</td>
+      <td>Sets the background color to green.</td>
     </tr>
     <tr>
-      <td class="sprk-u-FontWeight--bold">.sprk-u-BackgroundColor--gray-tint-50</td>
-      <td>Sets the background color to the gray tinted at 50%.</td>
+      <td class="sprk-u-FontWeight--bold">.sprk-u-BackgroundColor--green-dark</td>
+      <td>Sets the background color to dark green.</td>
     </tr>
     <tr>
-      <td class="sprk-u-FontWeight--bold">.sprk-u-BackgroundColor--gray-tint-25</td>
-      <td>Sets the background color to the gray tinted at 25%.</td>
+      <td class="sprk-u-FontWeight--bold">.sprk-u-BackgroundColor--green-tint-75</td>
+      <td>Sets the background color to green tinted at 75%.</td>
     </tr>
     <tr>
-      <td class="sprk-u-FontWeight--bold">.sprk-u-BackgroundColor--red-mid</td>
-      <td>Sets the background color to red-mid.</td>
+      <td class="sprk-u-FontWeight--bold">.sprk-u-BackgroundColor--green-tint-50</td>
+      <td>Sets the background color to green tinted at 50%.</td>
     </tr>
     <tr>
-      <td class="sprk-u-FontWeight--bold">.sprk-u-BackgroundColor--red-deep</td>
-      <td>Sets the background color to red-deep.</td>
+      <td class="sprk-u-FontWeight--bold">.sprk-u-BackgroundColor--green-tint-25</td>
+      <td>Sets the background color to green tinted at 25%.</td>
     </tr>
     <tr>
-      <td class="sprk-u-FontWeight--bold">.sprk-u-BackgroundColor--green</td>
-      <td>Sets the background color to green.</td>
+      <td class="sprk-u-FontWeight--bold">.sprk-u-BackgroundColor--red</td>
+      <td>Sets the background color to red.</td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">.sprk-u-BackgroundColor--red-tint-75</td>
+      <td>Sets the background color to red-tint-75.</td>
```

Comment:
This description should be like the other ones, for example, " green tinted at 50%.

Reply:
Forgot this one ^